app/views/layouts/_shim.html.erb

```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
```

Comment:
This looks like it is commented out?  If so, rather remove and put back in when needed

Reply:
Unless I am mistaken and it is actually doing something?

Reply:
This is html syntax for including an if check